.travis.yml

```diff
@@ -9,6 +9,7 @@ node_js:
   - '6'
 script:
   - yarn start validate
+  - yarn start validate.examples
```

Comment:
Awesome. If you could move this to `after_success` (after the `yarn start release` script), that would be great. That way the build doesn't break if the examples are broken. I just don't want a broken example to prevent someone's contribution making it into the lib or stopping a release. Thanks!